connector/src/main/resources/initial_data.json

```diff
@@ -36,101 +36,40 @@
         "light theme"
       ],
       "conformsTo": "conformsToSomething",
-      "creator": "Chuck Norris",
+      "creator": "Engineering Informatica S.p.A.",
       "description": [
         {
           "value": "Some text for multi language",
           "language": "en"
         }
       ],
-      "identifier": "Unique identifier for tests",
-      "issued": "yesterday",
-      "modified": "today",
-      "title": "Title for test",
+      "identifier": "Unique identifier for testing Catalog",
+      "issued": "4/23/24, 4:26 PM",
```

Comment:
We should use ISO Date Time Format yyyy-MM-dd'T'HH:mm:ss. SSSXXX

Reply:
For now, the ISO 8061 UTC is used (YYYY-MM-DDTHH:mm:ss.msZ). New ticket will be created for resolving this issue.